lib/dataset/test/bins_test.cpp

```diff
@@ -464,6 +464,7 @@ TEST_F(DatasetBinsTest, concatenate) {
   check();
   buffer0.setData("b", column * column);
   check_fail();
+  // cppcheck-suppress duplicateExpression  # Intentional, see above.
```

Comment:
What is duplicate here (in the eyes of cppcheck)?

Reply:
Because of `column / column` which is always 1-like.